releases/release-1.13/release_team.md

```diff
@@ -0,0 +1,13 @@
+| **Role** | **Name** (**GitHub / Slack ID**)  | **Shadow Name(s) (GitHub / Slack ID)** |
+| ------ | ------ | ------ |
+| Lead | Aishwarya Sundar ([@AishSundar](https://github.com/AishSundar)) | Aaron Crickenberger ([@spiffxp](https://github.com/spiffxp)) |
+| Features | Kendrick Coleman ([@kacole2](https://github.com/kacole2)) | Michael Singh ([@wadadli](https://github.com/wadadli)) |
+| CI Signal | | |
```

Comment:
Suggesting Guinevere Saenger (@guineveresaenger/ @gsaenger on Slack) for CI Shadow as per GH issue comment

Reply:
@AishSundar -- She also mentioned interest in Features Shadow, so I left her in the meantime, so I could follow up. I'll put her up as soon as I check in with her! :)